tests/test_calibrateImage.py

```diff
@@ -595,6 +616,36 @@ def test_runQuantum_no_optional_outputs(self):
             # Restore the one we removed for the next test.
             connections[optional] = temp
 
+    def test_runQuantum_no_calibrate_pixels(self):
```

Comment:
Should note in the test docstring that this is testing that the `applied_photo_calib` output has been removed.